src/pages/AppDetail.svelte

```diff
@@ -49,7 +50,11 @@
       <td><code>{app.app_id}</code></td>
     </tr>
   </table>
-
+  {#if app.prototype}
+    <AppAlert
+      message="The {params.app} application is a prototype. The metrics and pings listed below may contain inconsistencies and testing strings."
+      bgColor="#808895" />
+  {/if}
```

Comment:
This seems unrelated to this change?

Reply:
I did not add this. I only added the else part for this PR. 
I am confused why it has come into my edit. :sweat_smile:

Reply:
The formate changed after I ran `npm run format`. 
Otherwise, I was getting error on CircleCI for not running Prettier.